alcs-frontend/src/app/shared/details-header/details-header.component.ts

```diff
@@ -30,6 +40,7 @@ export class DetailsHeaderComponent {
   @Input() types: ApplicationTypeDto[] = [];
   @Input() days = 'Calendar Days';
   @Input() showStatus = false;
+  @Input() parentType: PARENT_TYPE = PARENT_TYPE.APPLICATION;
```

Comment:
idea: Can you make the input the status service and then you won't need the if else below?
@Input() statusService: NoticeOfIntentSubmissionStatusService | ApplicationSubmissionStatusService?

Reply:
Updated!